delivery_sending/models/sending_request.py

```diff
@@ -0,0 +1,147 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# Copyright 2022 Impulso Diagonal - Javier Colmeiro
+# Copyright 2022 Tecnativa - Víctor Martínez
+import logging
+
+from suds.client import Client
+from suds.sax.text import Raw
```

Comment:
Wasn't `zeep` suitable for the task?

Reply:
The current behavior works correctly. If there was a test user for the API I would change it to `zeep`.

Reply:
Ok, I didn't notice it was coauthored